### ISSUE TITLE
[Accessibility Issue]: Narrator tool in Windows 11 is not able to read content there in RichTextBox control of WPF application.

### ISSUE THREAD

**Issue by vedeevi** [dotnet/wpf#6771]
* .NET Framework Version: 4.7.2
* Windows version: (Windows 11 21H2 (Build : 22000.739)`)
* Does the bug reproduce also in WPF for .NET Framework 4.7.2?: Yes
* Is this bug related specifically to tooling in Visual Studio (e.g. XAML Designer, Code editing, etc...)? If yes, please file the issue via the instructions [here](https://docs.microsoft.com/visualstudio/ide/how-to-report-a-problem-with-visual-studio?view=vs-2019). No
* Security issues and bugs should be reported privately, learn more via our [responsible disclosure guidelines](https://github.com/dotnet/wpf/blob/main/README.md#reporting-security-issues-and-security-bugs).

 
 
 **Problem description:**
 Narrator tool in Windows 11 operating system is not able to read the text content available in paragraphs under RichTextBox control of WPF. However, this same text there in RichTextBox control is read by Narrator tool in Windows 10 21H2 OS.

Sample code for RichTextBox control used in our WPF application:
`<RichTextBox x:Name="AppContentRichTextBox" BorderThickness="0"
                                   x:FieldModifier="public" IsDocumentEnabled="True" IsReadOnly="True"
                                   FontSize="16" Foreground="Black" FontFamily="Segoe UI" 
                                     KeyboardNavigation.TabNavigation="Continue" Focusable="True"
                                   Margin="-5,0,0,30" AutomationProperties.Name="App content details"
                                   Visibility="Visible" HorizontalContentAlignment="Stretch" VerticalAlignment="Center">
                            <FlowDocument>
                                <Paragraph    KeyboardNavigation.TabNavigation="Continue" Focusable="True">
                                    This is the 
                                    <Bold> paragraph</Bold> text of the application which need to be read by 
                                    <Bold> Narrator tool </Bold> available in Windows 10 or 11 operating system.
                               </Paragraph>
                            </FlowDocument>
</RichTextBox>`

 **Actual behavior:** 
 No exceptions. 
 Narrator tool in Windows 11 is not able to read content there in RichTextBox control of WPF application.

 **Expected behavior:**
Narrator tool in Windows 11 has to read content there in RichTextBox control of WPF application. 

 **Minimal repro:**

**Comment by vedeevi** [comment]
Hi Team,

It's been 15 days since this issue has been posted, but still didn't seen update... please let us know when we can expect the reply for this issue. 

Thanks,
Vedeevi.

**Comment by pchaurasia14** [comment]
Hi @vedeevi, the issue is under investigation. We'll let you know once we have more information on this.

**Comment by singhashish-wpf** [comment]
@vedeevi 
It seems that the richtext box content is being read only when you enable the scan mode (Caps + Space) and then focus to the richtextbox, After than down arrow key. and the narrator would start reading the contents of the richtext box.

**Comment by anjalisheel-wpf** [comment]
@vedeevi 

Ashish has given you solution to read the text, check the link below for similar git issue : 

https://github.com/dotnet/wpf/issues/5035
